From da7a437a1f8645f5e910c0eb5f45f4c2ab8bc861 Mon Sep 17 00:00:00 2001
From: Harshesh Valera <harshesh.v@variscite.com>
Date: Wed, 21 Sep 2022 21:52:03 -0700
Subject: [PATCH 08/11] Revert "imx8mp_var_dart: Enable bluetooth wifi
 properties to be set"

This reverts commit bcf19e4f35a843fd0a548b60200bb1bfb71b9a48.
---
 prebuilts/api/31.0/private/property.te | 22 ++++++++++++++++++----
 private/property.te                    | 22 ++++++++++++++++++----
 2 files changed, 36 insertions(+), 8 deletions(-)

diff --git a/prebuilts/api/31.0/private/property.te b/prebuilts/api/31.0/private/property.te
index 74d144459..587cf5e2f 100644
--- a/prebuilts/api/31.0/private/property.te
+++ b/prebuilts/api/31.0/private/property.te
@@ -167,8 +167,6 @@ compatible_property_only(`
     -nfc_prop
     -powerctl_prop
     -radio_prop
-    -wifi_prop
-    -bluetooth_prop
   }:property_service set;
 
   neverallow {
@@ -199,6 +197,15 @@ compatible_property_only(`
     radio_prop
   }:property_service set;
 
+  neverallow {
+    domain
+    -coredomain
+    -bluetooth
+    -hal_bluetooth_server
+  } {
+    bluetooth_prop
+  }:property_service set;
+
   neverallow {
     domain
     -coredomain
@@ -219,6 +226,15 @@ compatible_property_only(`
     exported_camera_prop
   }:property_service set;
 
+  neverallow {
+    domain
+    -coredomain
+    -hal_wifi_server
+    -wificond
+  } {
+    wifi_prop
+  }:property_service set;
+
   neverallow {
     domain
     -init
@@ -272,7 +288,6 @@ compatible_property_only(`
     -coredomain
     -bluetooth
     -hal_bluetooth_server
-    -vendor_init
   } {
     bluetooth_prop
   }:file no_rw_file_perms;
@@ -282,7 +297,6 @@ compatible_property_only(`
     -coredomain
     -hal_wifi_server
     -wificond
-    -vendor_init
   } {
     wifi_prop
   }:file no_rw_file_perms;
diff --git a/private/property.te b/private/property.te
index 74d144459..587cf5e2f 100644
--- a/private/property.te
+++ b/private/property.te
@@ -167,8 +167,6 @@ compatible_property_only(`
     -nfc_prop
     -powerctl_prop
     -radio_prop
-    -wifi_prop
-    -bluetooth_prop
   }:property_service set;
 
   neverallow {
@@ -199,6 +197,15 @@ compatible_property_only(`
     radio_prop
   }:property_service set;
 
+  neverallow {
+    domain
+    -coredomain
+    -bluetooth
+    -hal_bluetooth_server
+  } {
+    bluetooth_prop
+  }:property_service set;
+
   neverallow {
     domain
     -coredomain
@@ -219,6 +226,15 @@ compatible_property_only(`
     exported_camera_prop
   }:property_service set;
 
+  neverallow {
+    domain
+    -coredomain
+    -hal_wifi_server
+    -wificond
+  } {
+    wifi_prop
+  }:property_service set;
+
   neverallow {
     domain
     -init
@@ -272,7 +288,6 @@ compatible_property_only(`
     -coredomain
     -bluetooth
     -hal_bluetooth_server
-    -vendor_init
   } {
     bluetooth_prop
   }:file no_rw_file_perms;
@@ -282,7 +297,6 @@ compatible_property_only(`
     -coredomain
     -hal_wifi_server
     -wificond
-    -vendor_init
   } {
     wifi_prop
   }:file no_rw_file_perms;
-- 
2.25.1

